.changes/unreleased/tinymce-TINY-10808-2024-04-02.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: Dialog title not being announced VoiceOver MacOS, dialog contains `aria-label`
+  instead of `aria-labelledby` on MacOS.
```

```suggestion
body: Dialog title was not announced in macOS VoiceOver, dialogs now use `aria-label`
 instead of `aria-labelledby` on macOS.
```

And similarly for the alloy changelog (although you can make it more technical, only the tinymce changelog needs to be customer-focused)